certified-connectors/GMO Sign/apiDefinition.swagger.json

```diff
@@ -0,0 +1,907 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "x-ms-deployment-version": "1.0.0",
+    "version": "1.0",
+    "title": "GMO Sign",
+    "description": "GMO Sign Connector helps organizations connect and automate how they prepare, sign, act on, and manage agreements.",
+    "x-ms-api-annotation": {
+      "status": "Production"
+    },
+    "contact": {
+      "name": "GMO Global Sign",
+      "url": "https://www.gmosign.com/"
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://secure.gmosign.com/"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://www.gmogshd.com/privacy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Content and Files;Productivity"
+    }
+  ],
+  "host": "y4omo61inf.execute-api.ap-northeast-1.amazonaws.com",

```

Comment:
@GMO-GSHD-APIGW can you please confirm that this host URL is for production environment? Using test or staging environment URLs is not allowed.